### ISSUE TITLE
Pitch based AP limiting.

### ISSUE THREAD

**Issue by Aquilux** [AndyMt/GravityTurn#44]
In some cases launch parameters require an assent throttle setting minimum that pushes AP too high while not allowing for engine shutdown. In other cases a more realistic launch may be desired. To accommodate this, I request an optional mode that pitches down to limit AP when throttle would otherwise have been brought down below throttle limit similar to pitching up to increase AP when throttle would have gone over 100%. 

PE limit would also be desirable to move on to coast once desired PE has been achieved, allowing for mostly spent stage to be left in suborbital trajectory.

Requested:
- [ ] AP limiting via pitch toggle.
- [ ] PE max limit value text box.
- [ ] Stage once after PE limit reached, AP achieved, and above atmo toggle.

**Comment by AndyMt** [comment]
I have some concerns in case solid boosters are involved or TWR is quite high. Then the vehicle could pitch down too much while still in thick atmosphere. I've tried this because I noticed the original "don't pitch down" code and removed it to see what happens. It got quite unpredictable, stuff broke up all the time, so I added the code again. Maybe with adding more parameters like an additional pressure threshold this could be solved, but this requires more time for investigations. Time I don't have at the moment.

**Comment by Bezzier** [comment]
Why not use an angle of attack limiter to keep the vehicle from pitching down too far?

**Comment by Bezzier** [comment]
Also, surely through simulation, we can discover the relationship between sail area, rocket height, speed, atmosphere thickness, and what counts as an overpitch.

**Comment by AndyMt** [comment]
Yes sure, but then... what is "too far"? As I said - right now I don't have time for it. If we are lucky I can invest a few hours next week, I have a few days off. But need most of that for Christmas preparations :)
